packages/node_modules/@webex/plugin-meetings/test/unit/spec/members/request.js

```diff
@@ -120,6 +122,38 @@ describe('plugin-meetings', () => {
     });
 
     describe('#lowerAllHands', () => {
+      const parameterErrorMessage = 'requestingParticipantId must be defined, and the associated locus url for this meeting object must be defined.';
+
+      const checkInvalid = async (functionParams) => {
+        assert.throws(() => membersRequest.lowerAllHandsMember(functionParams), ParameterError, parameterErrorMessage);
+        assert(membersRequest.request.notCalled);
+        assert(membersUtil.getLowerAllHandsMemberRequestParams.notCalled);
+      };
+
+      it('rejects if no options are passed in', async () => {
+        checkInvalid();
+      });
+
+      it('rejects if no locusUrl are passed in', async () => {
+        checkInvalid({requestingParticipantId: 'test'});
+      });
+
+      it('rejects if no requestingParticipantId are passed in', async () => {
+        checkInvalid({locusUrl: 'test'});
+      });
+
+      it('returns a promise', async () => {
+        const locusUrl = url1;
+        const memberId = 'test1';
+
+        const options = {
+          requestingParticipantId: memberId,
+          locusUrl,
+        };
+
+        await assert.becomes(membersRequest.lowerAllHandsMember(options), {});
```

Comment:
nitpick: It would be good to check that the promise returned was the same one returned by the call to `this.request` (since resolving to an empty object doesn't really tell us much), but not too concerned. This would probably be easier to do as a part of the test following this one (`'sends a PATCH to the locus endpoint'`).